src/sentry/relocation/tasks/transfer.py

```diff
@@ -82,7 +90,86 @@ def _find_relocation_transfer(
     silo_mode=SiloMode.CONTROL,
 )
 def process_relocation_transfer_control(transfer_id: int) -> None:
-    pass
+    log_context = {"id": transfer_id, "silo": "control"}
+    try:
+        transfer = ControlRelocationTransfer.objects.get(id=transfer_id)
+    except ControlRelocationTransfer.DoesNotExist:
+        logging.warning("relocation.transfer_missing", extra=log_context)
+        return
+    log_context["state"] = transfer.state
+    log_context["relocation_uuid"] = str(transfer.relocation_uuid)
+
+    if transfer.state == RelocationTransferState.Request:
+        public_key = transfer.public_key or b""
+        if public_key:
+            public_key = bytes(public_key)
+
+        # Forward the export request to the exporting region.
+        try:
+            region_relocation_export_service.request_new_export(
```

Comment:
Do our RPCs have a maximum timeout before they'll fail? I'm wondering if this will become another pain point in the export process.

Reply:
Yeah, 90s is the max I think, and yes it could be a future pain point for larger exports. I don't have a solution for that yet.